packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -144,7 +144,7 @@ function mouseDownListener(evt: MouseEvent) {
 
   state.element = <HTMLDivElement>evt.currentTarget;
 
-  state.mouseButton = evt.button;
+  state.mouseButton = evt.buttons;
```

Comment:
this is something that might be used to get multi buttons at the same time. But depending on how user press them we might have different situations. I.e 
1. left then right then move
1. right then left then move
1. right then move a bit then left then move

Reply:
There absolutely are different setups, so it only works for interactive tools like Pan, and not click based tools like annotations.  There is no distinction between the various click options if you only consider tools that listen to move.